### PR TITLE
Add `gunichar` support with scalar conversion

### PR DESCRIPTION
**Pull request by gkoz**

### CODE REVIEW

**Comment by EPashkin** [comment]
`gtk_entry_set_invisible_char` now work, but IMHO needed nullable override from #176.
And `gtk_entry_get_invisible_char` also needed unrealized return nullable override.

**Comment by gkoz** [comment]
I hadn't thought of applying nullability to ints. This could work. You're right that it's more consistent to use `Option` here, but then it's far easier to just implement these two functions manually (because `char` can't just be compared to `0`).

We might want to work out a generator solution for harmonization of ints (as in `EntryBuffer`, `Notebook` and `invisible_char`) but feasibility is unclear.

**Comment by gkoz** [comment]
Okay, this isn't that difficult, it's the conversion functions that take care of comparisons etc. I'll try the generator way.

**Comment by gkoz** [comment]
This should settle the `get/set_invisible_char` case.

File: src/config/functions.rs [hunk 1]
```diff
@@ -257,27 +256,34 @@ name = "par1"
...
 const = false
+nullable = false
 [[parameter]]
 name = "par3"
 const = true
+nullable = true
 [[parameter]]
 pattern = "par4"
 const = true
+nullable = true
```
Comment (EPashkin) [inline src/config/functions.rs]: Can you change this to None?

Comment (gkoz) [inline src/config/functions.rs]: I don't follow. `None` corresponds to absence of this line. Remove it?

Comment (EPashkin) [inline src/config/functions.rs]: Oh, yes, remove.

**Comment by gkoz** [comment]
Updated

**Comment by EPashkin** [comment]
Tnx

**Comment by gkoz** [comment]
@homu r+

**Pull request merged**